tests/fatrace-comm-json

```diff
@@ -0,0 +1,50 @@
+#!/bin/sh
+set -euC
+
+mkdir -m 777 tmp
+trap "rm -rf tmp" EXIT INT QUIT PIPE
+
+LOG="$AUTOPKGTEST_TMP/fatrace.log"
+echo "starting fatrace --command touch..."
+fatrace --current-mount --command touch -s 2 -o $LOG --json &
+sleep 1
+
+echo "create files with different programs"
+touch tmp/includeme
+dd if=/dev/zero of=tmp/notme bs=1 count=1
+
+echo "waiting for fatrace..."
+wait
+
+echo "checking log..."
+check_log() {
+    if ! jq < $LOG >/dev/null; then
+        echo "Illegal JSON" >&2
```

Comment:
Missing `exit 1`? Or rather, isn't this whole call redundant? The jq call below would already complain on invalid JSON syntax.